testRNN/src/mnistClass.py

```diff
@@ -8,12 +8,11 @@
 import os
 import numpy as np
 import copy
-from utils import getActivationValue,layerName, hard_sigmoid, get_activations_single_layer, \
-    setup_dir_for_file
+from utils_testRNN import getActivationValue,layerName, hard_sigmoid, get_activations_single_layer
 
 
 class mnistclass:
-    def __init__(self, datasetName, modelFile):
+    def __init__(self,modelName):
```

Comment:
This change reverts some recent changes to the master branch. This is a hint that you need to pull the master branch before creating the pull request: you may then be asked by git to merge those more recent changes to the master branch into your own changes (if you're lucky this will be automated and you won't have to do anything manually).

Reply:
PyQt5 and runipy is needed. you can install them by command below: 
pip install PyQt5
pip install runipy

and launch the GUI by command below:
runipy GUI.ipynb

Reply:
Thanks for the dependencies and commands. Note I encountered many issues in trying to use `runipy`, and found out it is quite an out-dated tool. So I ended up searching a bit for alternatives, and used `jupyter` with the command below:
```sh
jupyter nbconvert --to python GUI.ipynb --stdout | python3
```
(I don't know what Python package the `jupyter` command comes from. I just happened to have it installed already.)
So I managed to run it all as it is.

However, some changes introduced in commit 3be04847c08cff5cc64ad8b4553f0ef49b5a276d reverts some recent changes. I think cleaning that up properly requires rebasing on your own side. Although, to keep the process simple enough, you may be able to revert it manually with `git show 3be04847c08cf | git apply -R`, and then you commit only the changes you did not do yourself (with `git commit -p`).  After that you `git stash` (which should put your own modifications back in place), and you shall be able to test that the tool still works.

If that's too intricate, just tell me; I should be able to do this on my side at some point (I did most or all of the changes in master that we need to re-introduce).